arguments.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import argparse
+
+
+def parameters():
+    parser = argparse.ArgumentParser(
+            prog='BetterWeather',
+            description="""This program is written in Python.
+            Its purpose is to collect weather forecast data
+            from a few free weather forecast APIs (e.g. Open-Meteo, WeatherAPI)
+            and combine it into one more accurate weather prediction.
+            After providing the location eg. "Madrid" you will receive a file
+            with weather data and a user-friendly plot.""")
+
+    # positional arguments
+    parser.add_argument('location', metavar='location', type=str,
+                        help="""Enter the location for which you want
+                         to get the weather forecast e.g. "Cracovia".""")
+
+    # optional arguments
+    parser.add_argument('-fp', metavar='forecast_period', type=str,
+                        choices=['24h', '48h', '72h', '7days', '10days', '14days'],
+                        help="""Enter the forecast period you are interested
+                         in. Valid options are: 24h, 48h, 72h, 7days, 10days,
+                         14days (default: 24h).""", default='24h')
```

Comment:
In the FD [I see more arguments](https://docs.google.com/document/d/1BwLEyIXszuNKcbiCdtWt22Z8ZKOX6ePOV2JH_O4gA0I/edit#heading=h.fkzs10gyuvl1) than you're handling here.

Reply:
True. I want to develop this and other components when the program will give the expected output for the basic parameters. So for now it is ok and it will be continued.